src/guess/stringIndices.js

```diff
@@ -0,0 +1,10 @@
+function stringIndices( input , position ) {
+  let onlyLetters = input.replace(/[!@#$%^&*,]/g, ""),
+      splitString = onlyLetters.split(" ")
+
+      if ( isNaN(position) || position > splitString.length || position < 0 ) {
+        return ("Empty")
```

Comment:
The parentheses here are a little confusing - any time my eye scans over something like this, I interpret it first as a function call.  Consider omitting parentheses for clarity.